xscen/catalog.py

```diff
@@ -22,14 +22,27 @@
 from intake.source.utils import reverse_format
 from intake_esm.cat import ESMCatalogModel
 
-from .common import CV
 from .config import CONFIG, parse_config, recursive_update
 from .io import get_engine
+from .utils import CV
 
 logger = logging.getLogger(__name__)
 # Monkey patch for attribute names in the output of to_dataset_dict
 intake_esm.set_options(attrs_prefix="cat")
 
+__all__ = [
+    "COLUMNS",
+    "ESMCatalogModel",
+    "ID_COLUMNS",
+    "concat_data_catalogs",
+    "date_parser",
+    "generate_id",
+    "parse_config",
+    "parse_directory",
+    "parse_from_ds",
```

Comment:
```suggestion
    "ID_COLUMNS",
    "concat_data_catalogs",
    "date_parser",
    "generate_id",
    "parse_directory",
    "parse_from_ds",
```

`ESMCatalogModel` and `parse_config` are not declared here, doesn't that mean they shouldn't be in the `__all__`?

Reply:
Very true. Thanks!